apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/wp-interface-types.d.ts

```diff
@@ -1,3 +0,0 @@
-declare module '@wordpress/interface' {
```

Comment:
Should we create a corresponding `wp-edit-post-types` file to carry this over for `@wordpress/edit-post`?

Reply:
Oh yes, thanks for pointing this out. I originally removed this file because I assumed that all `@wordpress/*` packages now ship with types. But only some do. Now I addded a module definition for `@wordpress/edit-post` and it immediately fixed TypeScript errors on imports that were reported in the editor IDE.